src/gui/MainWindow.cpp

```diff
@@ -308,12 +308,13 @@ void MainWindow::finalize()
 					SLOT( exportProjectTracks() ),
 					Qt::CTRL + Qt::SHIFT + Qt::Key_E );
 
-	// temporarily disabled broken MIDI export				
-	/*project_menu->addAction( embed::getIconPixmap( "midi_file" ),
+#ifdef LMMS_HAVE_FLUIDSYNTH
```

Comment:
@tonychee7000, let's not discuss that plugin here. There's issue #793 and possibly others for that.